core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -554,7 +561,7 @@ public JSONArray getPlatformPluginUpdates() {
         JSONArray initialPluginList = null;
         updateSiteList: for (UpdateSite updateSite : Jenkins.get().getUpdateCenter().getSiteList()) {
             String updateCenterJsonUrl = updateSite.getUrl();
-            String suggestedPluginUrl = updateCenterJsonUrl.replace("/update-center.json", "/platform-plugins.json");
+            String suggestedPluginUrl = updateCenterJsonUrl.replace("/update-center.json", SUGGESTED_PLUGINS_FILE_NAME);
```

Comment:
given this needs co-operation from the actual update site to provide the file, would it not be better to obtain the URL from the `UpdateSite`  (introduce a new method that does the same as today and then anyone that needs to override it can do so)  (after all if the Jenkins project decides to offer a different list of recommended plugins in FIPS mode, I would expect that to be driven by the FIPS flag and the UpdateSite make a switch, and a user would not need to provide 2 switches).

This also only works for a hosted (http(s)) provider and the [fallback does not use the property](https://github.com/jenkinsci/jenkins/blob/569c8b0f7fb5327a7d3ea832359c8ad7aa2ea9b8/core/src/main/java/jenkins/install/SetupWizard.java#L621-L631)

Reply:
yup sounds better to have only one flag to use and avoid adding too many system properties.
I changed to add a method to UpdateSite